spotlight-client/src/contentModels/HistoricalPopulationBreakdownMetric.ts

```diff
@@ -196,4 +199,44 @@ export default class HistoricalPopulationBreakdownMetric extends Metric<Historic
 
     return countsByDate.length ? countsByDate : undefined;
   }
+
+  get readme(): string {
+    const unknowns = this.unknowns;
+    if (!unknowns) {
+      return this.methodology;
+    }
+
+    const formattedUnknowns = unknowns
+      .map((entry) => {
+        const parts: string[] = [];
+
+        DemographicFieldKeyList.forEach((key) => {
+          const value = entry.unknowns[key];
+          if (!value) return;
+
+          parts.push(
+            `${getDemographicViewLabel(key).toLowerCase()} (${formatAsNumber(
+              value
+            )})`
+          );
+        });
+        const formattedCounts = parts.join(", ");
+        let label: string;
+
+        label = format(entry.date, "MMM d y");
+
+        return `${formattedCounts} for ${label}`;
+      })
+      .join("; ");
+
+    return (
+      this.methodology +
+      `\n\nVISUALIZATION FOOTNOTES\n\n
```

Comment:
in `downloadData` we have this step where we take the readme and strip out newlines and replace them with spaces. I'm thinking we should format our readme so we don't have to do that, so we can have newlines in our readme?

Reply:
I agree this seems like the right place to do that preprocessing, if you can figure out a good solution!

Reply:
I strip away the newlines in the `methodology` but preserve newlines in my visualization footnotes.